src/admin/navigation/navigationItemModal.tsx

```diff
@@ -624,7 +624,16 @@ export class NavigationItemModal extends React.Component<NavigationItemModalProp
                             label="Select method"
                             options={urlTypeOptions}
                             selectedKey={this.state.selectedUrlType}
-                            onChange={(event, option) => this.setState({ selectedUrlType: option.key })}
+                            onChange={(event, option) => {
+                                let errors = this.state.errors;
+                                if (option.key.toString() === LinkOptionKey.SavedUrl) {
```

Comment:
Can option (and key) parameter undefined or null?

Reply:
No. It is a radio select with two options and one of them is always selected by default.